tests/integration_tests/test_login.py

```diff
@@ -0,0 +1,22 @@
+from .pages import LoginPage  # noqa isort:skip
```

Comment:
No longer necessary since the imports are no longer in the middle of the file
```suggestion
from .pages import LoginPage
```